core/server/models/base/listeners.js

```diff
@@ -8,7 +8,10 @@ var moment = require('moment-timezone'),
  * WHEN access token is created we will update last_seen for user.
  */
 common.events.on('token.added', function (tokenModel) {
```

Comment:
Suggestion: This logic is only used for v0.1. IMO add a TODO to remove when we drop v0.1?